benchmarks/oss.py

```diff
@@ -104,7 +104,10 @@ def train(
         optimizer = model.sharded_optimizer
 
     else:
-        model = DDP(model, device_ids=[rank], find_unused_parameters=False)  # type: ignore
+        if args.cpu:
+            model = DDP(model, device_ids=None, find_unused_parameters=False)  # type: ignore
```

Comment:
perhaps conditionally set the device_ids's value to a variable and avoid duplicating the entire model = DDP() line in the if and else cases?

Reply:
Sure. This is done in the second commit